src/main/res/layout/secondary_action_list_item.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Like we talked about, let's not use includes. It complicates the setup of these lists by the caller and the idea is to make it as simple as possible